### DISCUSSION TITLE
Specifying HTTP feed as NuGetSources

### DISCUSSION THREAD

**Discussion by YuliiaKovalova** [NuGet/Home#12484] [General]
Hi Team,

Could you tell me if you validate somehow on your side if HTTP vs HTTPS feed was passed as `NuGetSources` param?
Thank you!

**Comment by zivkan** [comment]
`NuGet.Configuration.PackageSource` has [`IsHttp` and `IsHttps` properties](https://github.com/NuGet/NuGet.Client/blob/fc41cff354547a3c7f84d88d10d4e0bdf79d9c4b/src/NuGet.Core/NuGet.Configuration/PackageSource/PackageSource.cs#L99-L107).

Note as the XMLDoc says for `IsHttp`, it returns true if it's using either unencrypted HTTP, or encrypted HTTPS. This is because the `IsHttps` property was only added recently (in the context of NuGet's lifetime), and the behaviour of `IsHttp` was not changed, to avoid a breaking change.
